packages/block-library/src/playlist/edit.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External Dependencies
+ */
+
+import { uniqBy } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	FormFileUpload,
+	IconButton,
+	Placeholder,
+	ServerSideRender,
+	Toolbar,
+	withNotices,
+	CheckboxControl,
+	PanelBody,
+	SelectControl,
+} from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import {
+	BlockControls,
+	InspectorControls,
+	mediaUpload,
+	MediaUpload,
+} from '@wordpress/editor';
+
+class PlaylistEdit extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.initializePlaylist = this.initializePlaylist.bind( this );
+		this.uploadFromFiles = this.uploadFromFiles.bind( this );
+		this.onUploadFiles = this.onUploadFiles.bind( this );
+
+		// check for if ids is set to determine edit state
+		this.state = {
+			isEditing: ! this.props.attributes.ids,
+		};
+	}
+
+	initializePlaylist() {
+		window.wp.playlist.initialize();
```

Comment:
Given the behavior of `ServerSideRender` to trigger this on any update to the response, do we need to consider the fact that the rendered preview could be updated multiple times? Are we leaving [zombie event handlers](https://lostechies.com/derickbailey/2011/09/15/zombies-run-managing-page-transitions-in-backbone-apps/) by not cleaning up after the old content? I'm not entirely sure how `wp.playlist.initialize` is intended to work here.

Reply:
This is needed because the initialization needs to happen after the editor enqueues the dependencies listed in client-assets here: https://github.com/WordPress/gutenberg/pull/9169/files#diff-620130744b6b73b822a3df609671d930

A playlist will not render correctly (using mejs) unless the initialize happens at this point. I worked a bit with @noisysocks on this until we determined this is where the initialization should be called to fix the render